src/test/scala/edu/berkeley/cs/rise/opaque/OpaqueOperatorTests.scala

```diff
@@ -389,6 +389,22 @@ trait OpaqueOperatorTests extends FunSuite with BeforeAndAfterAll { self =>
     df.filter($"word".contains(lit("1"))).collect
   }
 
+  testAgainstSpark("concat with string") { securityLevel =>
```

Comment:
Add a test case for empty string.

Reply:
combined with "concat with other datatype test"